<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelSettingsDialog</class>
 <widget class="QDialog" name="ModelSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beállítások</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Beállítások</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="showStepsCheckBox">
        <property name="text">
         <string>Megoldások levezetésének mutatása</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="coordinateScaleableCheckBox">
        <property name="text">
         <string>Koordináta rendszerek skálázhatóak</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="useWolframAlphaCheckBox">
        <property name="text">
         <string>WolframAlpha használata</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showMonterayCheckBox">
        <property name="text">
         <string>Monetáris ablak mutatása</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showFiscalCheckBox">
        <property name="text">
         <string>Fiskális ablak mutatása</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModelSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModelSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
